ansible/roles/controller/tasks/deploy.yml

```diff
@@ -16,6 +16,27 @@
     mode: 0777
   become: "{{ logs.dir.become }}"
 
+- name: ensure controller config directory is created with permissions
+  file:
+    path: "{{ controller.confdir }}/controller{{ groups['controllers'].index(inventory_hostname) }}"
+    state: directory
+    mode: 0777
```

Comment:
Is 0777 necessary here?

Reply:
Same for the files above.

Reply:
Yes. File move operations need these rights.